Extensions/show_more.js

```diff
@@ -645,8 +618,8 @@ XKit.extensions.show_more = new Object({
 
 				var username = $(this).parent().attr('data-tumblelog-name');
 
-				if (XKit.extensions.show_more.submit_available[username] === true ||
-					XKit.extensions.show_more.submit_available[username] === false) {
+				if (XKit.extensions.show_more.submit_available[username] ||
+					!XKit.extensions.show_more.submit_available[username]) {
```

Comment:
Won't this always return? This question applies to likes_available and anon_available below.

Before this change it would only return if the properties were booleans, but I haven't looked into whether they could be anything else.